src/card-info.css

```diff
@@ -0,0 +1,129 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+}
+
+.card__img--position {
+  margin: 32px 19px 40px;
```

Comment:
just add paddings to card container, and you don't need to add it to image and `card__item`